providers/aws/resources/aws_iam.go

```diff
@@ -288,7 +288,9 @@ func (a *mqlAwsIam) virtualMfaDevices() ([]interface{}, error) {
 
 	devicesResp, err := svc.ListVirtualMFADevices(ctx, &iam.ListVirtualMFADevicesInput{})
 	if err != nil {
-		return nil, errors.Wrap(err, "could not gather aws iam virtual-mfa-devices")
```

Comment:
any reason why we not set the error?

Reply:
ah ya we should do that, pushing the change now

Reply:
With https://github.com/mondoohq/cnquery/pull/3191 a plain error would also work. This approach is also fine.